src/modules/subRegion/SubRegions.ts

```diff
@@ -68,7 +69,7 @@ SubRegions.addSubRegion(Region.unova, new SubRegion('Unova', UnovaSubRegions.Uno
 SubRegions.addSubRegion(Region.kalos, new SubRegion('Kalos', KalosSubRegions.Kalos));
 
 SubRegions.addSubRegion(Region.alola, new SubRegion('Melemele Island', AlolaSubRegions.MelemeleIsland, undefined, 'Hau\'oli City'));
-SubRegions.addSubRegion(Region.alola, new SubRegion('Akala Island', AlolaSubRegions.AkalaIsland, new GymBadgeRequirement(BadgeEnums.FightiniumZ), 'Heahea City'));
+SubRegions.addSubRegion(Region.alola, new SubRegion('Akala Island', AlolaSubRegions.AkalaIsland, new GymBadgeRequirement(BadgeEnums.FightiniumZ), 'Heahea City')); // TODO: new QuestLineCompletedRequirement('Welcome to paradise, cousin!'), 'Heahea City'));
```

Comment:
Is this todo for when you do part two?

Reply:
This is todo because the quest currently is behind a development requirement and cannot be completed